get-started.html

```diff
@@ -0,0 +1,176 @@
+            <a href="https://mega.nz/#!YQElAYrQ!9CbQwoPHuIgyJoBvImGaejB-9qs8_EQaNnXXwiOfJtQ">Windows Wallet</a>
+            <br>
+            <a href="https://mega.nz/#!CMgAFLbJ!1kzPoI5UVNzsmrRPkr9fGWBFa6ecU0gZnLEACSYYaxs">Recent Blockchain</a>
+          </div>
+          <div class="step">
+            <h2>Step 2)</h2>
+            <h4><b>Encryption & Backup</b></h4>
+            <p class="explanation">Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptates, dolorum fugiat, reprehenderit odit sapiente quas distinctio perferendis tenetur exercitationem placeat minus facilis aspernatur, doloribus nostrum quae cum culpa mollitia laudantium!</p>
+          </div>
+          <div class="step">
+            <h2>Step 3)</h2>
+            <h4><b>Sub Title</b></h4>
+            <p class="explanation">Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptates, dolorum fugiat, reprehenderit odit sapiente quas distinctio perferendis tenetur exercitationem placeat minus facilis aspernatur, doloribus nostrum quae cum culpa mollitia laudantium!</p>
+          </div>
+          <div class="step">
+            <h2>Step 4)</h2>
+            <h4><b>Sub Title</b></h4>
+            <p class="explanation">Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptates, dolorum fugiat, reprehenderit odit sapiente quas distinctio perferendis tenetur exercitationem placeat minus facilis aspernatur, doloribus nostrum quae cum culpa mollitia laudantium!</p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+  
+  <!--==========================
+  Footer
+============================-->
+  <footer id="footer">
+    <div class="container">
+      <div class="row">
+        <div class="col-md-12">
+          <div class="copyright">
+            &copy; Copyright
+            <strong>BUZZCOIN PROJECT</strong>. All Rights Reserved
+          </div>
+          <div class="credits">
+            <!--
+                All the links in the footer should remain intact.
+                You can delete the links only if you purchased the pro version.
+                Licensing information: https://bootstrapmade.com/license/
+                Purchase the pro version with working PHP/AJAX contact form: https://bootstrapmade.com/buy/?theme=Imperial
+              -->
+            Bootstrap Theme originally created by
+            <a href="https://bootstrapmade.com/">BootstrapMade</a>, modified and extended by the BUZZ Project.
+          </div>
+        </div>
+      </div>
+    </div>
+  </footer>
+  <!-- #footer -->
+
+  <a href="#" class="back-to-top">
+    <i class="fa fa-chevron-up"></i>
+  </a>
+
+  <!-- Required JavaScript Libraries -->
+  <script src="vendor/jquery/jquery.min.js"></script>
```

Comment:
make sure you clear up any unused scripts here. A lot of the javascript is probably not in use.